packages/base-driver/lib/jsonwp-proxy/proxy.js

```diff
@@ -204,16 +204,18 @@ export class JWProxy {
         try {
           reqOpts.data = JSON.parse(body);
         } catch {
-          throw new Error(`Cannot interpret the request body as valid JSON: ${truncateBody(body)}`);
+          throw new Error(
+            `Cannot interpret the request body as valid JSON:`, logger.markSensitive(truncateBody(body))
```

Comment:
make sure there is a space after a colon
please split the single long line into two for better readability